### PR TITLE
Make changes to some formatting as well as change the part about the

### PR DESCRIPTION
**Pull request by RachelChua**
date as it is too lengthy.

### CODE REVIEW

**Review by dlimyy — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -236,16 +236,14 @@ Adds exam into exam list.
...
-* EXAM_DESCRIPTION should not be empty. Otherwise, an error message will be displayed to the user. 
-* EXAM_DATE should be in DD-MM-YYYY format, a valid date, and a date that is not earlier than today's date.
-  DD should be between 1 and 31(both inclusive) and MM should be between 1 and 12(both inclusive). Otherwise, an error message will be displayed to the user.
-  For example, 2022-12-28, 20-13-2022 are not in DD-MM-YYYY format, 30-02-2022 is an invalid date, and 20-08-2022 is a past date if the current date is 26-10-2022.
-
+* `MODULE` has to exist in the module list and be a valid module with at least 6 characters long with the first two being alphabetic characters. Otherwise, an error message will be displayed. 
+* `EXAM_DESCRIPTION` should not be empty. Otherwise, an error message will be displayed to the user. 
+* `EXAM_DATE` must be in the format `DD-MM-YYYY`, otherwise an error message will be shown. For example, 2022-12-28, 20-13-2022 are not in `DD-MM-YYYY` format
+* `EXAM_Date` must not be earlier than today's date, otherwise an error message will be shown. For example, 20-08-2022 is a past date if the current date is 26-10-2022.
```
Comment (Sampy147) [inline docs/UserGuide.md:246]: I think `EXAM_Date` should be capitalised i.e `EXAM_DATE`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -257,19 +255,17 @@ Edits the specified exam by updating the existing values to the input values.
...
-* EXAM_DESCRIPTION should not be empty. Otherwise, an error message will be displayed to the user.
-* EXAM_DATE should be in DD-MM-YYYY format, a valid date, and a date that is not earlier than today's date.
-DD should be between 1 and 31(both inclusive) and MM should be between 1 and 12(both inclusive). Otherwise, an error message will be displayed to the user.
-  For example, 2022-12-28, 20-13-2022 are not in DD-MM-YYYY format, 30-02-2022 is an invalid date, and 20-08-2022 is a past date if the current date is 26-10-2022.
-
+* `MODULE` has to exist in the module list and be a valid module with at least 6 characters long with the first two being alphabetic characters. Otherwise, an error message will be displayed.
+* `EXAM_DESCRIPTION` should not be empty. Otherwise, an error message will be displayed to the user. 
+* `EXAM_DATE` must be in the format `DD-MM-YYYY`, otherwise an error message will be shown. For example, 2022-12-28, 20-13-2022 are not in `DD-MM-YYYY` format
+* `EXAM_Date` must not be earlier than today's date, otherwise an error message will be shown. For example, 20-08-2022 is a past date if the current date is 26-10-2022.
```
Comment (Sampy147) [inline docs/UserGuide.md:268]: Same comment as the above

**Review by Sampy147 — COMMENTED** [review]
LGTM!

**Pull request merged**